Add links to some resources for the demo night

mgunyho opened the pull request:
@felixbade:
- is this a good idea to put these under platforms like this or should we make a separate section at the end?
- any other comment for the texts?
- could you test if the `<details>` looks ok when rendered, or do we need some extra CSS for it?

if not, feel free to merge

Commit:
Add links to some resources

apparently the </details> has to be without indentation, otherwise the list breaks

Commit:
Make <details> look clickable

mgunyho committed:
More links for TIC-80

felixbade commented:
It looks like this:

<img width="452" alt="Screenshot 2023-10-18 at 19 08 09" src="https://github.com/DOTAalto/dot-ry.fi/assets/6554998/83f18545-b130-4d86-8628-a48217fe9c1a">

And the HTML looks like this:

<img width="414" alt="Screenshot 2023-10-18 at 19 08 52" src="https://github.com/DOTAalto/dot-ry.fi/assets/6554998/c5d772bf-081e-471a-946c-376616964e7e">

Feel free to reopen this merge request if you are able to fix these.

mgunyho commented:
Wot, I'll need to look into it.

mgunyho committed:
Remove <details> stuff for now, reword

The <details> were stripped out by Hugo, would need to fiddle with it a bit to fix, can't be bothered to do it now

mgunyho commented:
I renamed "Platforms" to "Platforms, resources" and just have the links there as sublists, lemme know if it's not ok

![image](https://github.com/DOTAalto/dot-ry.fi/assets/20118130/464a4ce5-a58f-4c09-847b-632d2f1b7935)
